pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -124,31 +124,23 @@ let
     wrapperArgsStr = if lib.isString wrapperArgs then wrapperArgs else lib.escapeShellArgs wrapperArgs;
 
     generatedWrapperArgs = let
-      op = acc: normalizedPlugin: acc ++ normalizedPlugin.plugin.runtimeDeps or [];
-
-      runtimeDeps = lib.foldl' op [] pluginsNormalized;
-
-      binPath = lib.makeBinPath (
-           lib.optional finalAttrs.withRuby rubyEnv
-        ++ lib.optional finalAttrs.withNodeJs nodejs
-        ++ lib.optionals finalAttrs.autowrapRuntimeDeps runtimeDeps
-        );
-    in
-
-      # vim accepts a limited number of commands so we join them all
-          [
-            "--add-flags" ''--cmd "lua ${providerLuaRc}"''
-          ]
-          ++ lib.optionals (packpathDirs.myNeovimPackages.start != [] || packpathDirs.myNeovimPackages.opt != []) [
-            "--add-flags" ''--cmd "set packpath^=${finalPackdir}"''
-            "--add-flags" ''--cmd "set rtp^=${finalPackdir}"''
-          ]
-          ++ lib.optionals finalAttrs.withRuby [
-            "--set" "GEM_HOME" "${rubyEnv}/${rubyEnv.ruby.gemPath}"
-          ] ++ lib.optionals (binPath != "") [
-            "--suffix" "PATH" ":" binPath
-          ]
-          ;
+
+        binPath = lib.makeBinPath finalAttrs.runtimeDeps;
```

Comment:
Maybe just inline it now that it's only referenced once and it's short enough?

Reply:
done